README.md

```diff
@@ -1,51 +1,31 @@
-## [Landwind - Tailwind CSS Landing Page](https://demo.themesberg.com/landwind/)
+# techbrain-website
 
-<a href="https://demo.themesberg.com/landwind/" rel="Tailwind CSS Landing Page">![Tailwind CSS Landing Page Demo - Flowbite](https://themesberg.s3.us-east-2.amazonaws.com/public/github/landwind/thumbnail.png)</a>
+![Node.js Version](https://img.shields.io/badge/node-19.5.0-brightgreen)
```

Comment:
As mentioned in https://namespace-inc.atlassian.net/browse/TECHBRAIN-316. Please add `.node-version` by setting `nodenv local <version>`